fastlane/Fastfile

```diff
@@ -86,17 +87,17 @@ platform :ios do
   desc "Testflight build for Live Channel"
   lane :testflight_live do
 
-    updated_version_number_from_version_file = File.read("VERSION").gsub!(/\s+/, '')
+    udpated_version_number = udpate_version_in_xcode_proj(pbx_project_file, get_version_from_file)
 
-    # udpated_version_number = increment_version_number(
-    #   xcodeproj: project,
-    #   version_number: updated_version_number_from_version_file
-    # )
+    ci_run_build_number = ENV["CIRCLE_BUILD_NUM"]
 
-    udpated_version_number = updated_version_number_from_version_file
+    branch_commit_push_and_create_pr_if_needed(
+      version_number: udpated_version_number,
+      build_number: ci_run_build_number
+    )
 
     udpated_build_number = increment_build_number(
-      build_number: ENV["CIRCLE_BUILD_NUM"]
+      build_number: ci_run_build_number
```

Comment:
Does it make sense to update the build number of our project to `0` or `999` or something that shows somehow that this is dynamically set?

Reply:
Yeah good idea. No need to commit the build number back to the proj-file. Personally I'd favour 0 over 999